bazel/repositories.bzl

```diff
@@ -6,14 +6,6 @@ load("@com_google_googleapis//:repository_rules.bzl", "switched_rules_by_languag
 
 # dict of {build recipe name: longform extension name,}
 PPC_SKIP_TARGETS = {"luajit": "envoy.filters.http.lua"}
-NOBORINGSSL_SKIP_TARGETS = {
-    # The lua filter depends on BoringSSL
-    "lua": "envoy.filters.http.lua",
-
-    # These two extensions are supposed to be replaced with alternative extensions.
-    "tls": "envoy.transport_sockets.tls",
-    "tls_inspector": "envoy.filters.listener.tls_inspector",
```

Comment:
I suspect there might be symbol name conflicts when both BoringSSL and OpenSSL `tls` extensions are compiled in.

Reply:
That shouldn't happen with a custom extension configuration file. See this.

https://github.com/venilnoronha/envoy-openssl/blob/940988b0bd461965d466c86f2d86ec29fa0ec4c7/envoy_build_config/extensions_build_config.bzl#L73